README.md

```diff
@@ -39,11 +37,33 @@ The following requirements and principles has been used when designing the Elect
 
      *Example: Base45 encoding for optical transport.*
 
+## Trust model
```

Comment:
I've tried to work this in & also alleviate the concern a lot of folks will immediately have.

Or did you mean this as - no CSCA at all. Because I am not sure that that will fly with the other countries.

And that may well be an understatement.